src/JustEat.StatsD/UdpTransport.cs

```diff
@@ -29,7 +29,7 @@ public void Send(string metric)
             }
         }
 
-        public Socket CreateSocket()
+        private static Socket CreateSocket()
```

Comment:
👍 - I saw this during 2.x work, but didn't change it as it would've been a breaking change (and then forgot about it).